### DISCUSSION TITLE
How do we integrate human prior knowledge into our networks?

### DISCUSSION THREAD

**Discussion by bw4sz** [weecology/DeepForest#522] [Ideas]
A primary use case, especially for the bird monitoring arena, is replacing existing human or image-annotation surveys. Rather than throwing away the human and trying to directly slot in a machine learning model, we want to move towards a future of collaboration. Humans know alot of things that are auxiliary to the image pixels, the behavior, location, relative abundance of species, the differences in appearance through time. All of these things can be learned by an algorithm, but in the open set problem of novel datasets, we want to reduce the burden on annotation. Fully bayesian networks are probably still immature, but I can imagine a system that builds in 1) species distributions, for example using map of life or ebird. Lots of papers from the iNaturalist community that do this well. 2) Relative abundances and co-occurrence matrices. 3) Habitat descriptions. These pieces of data could be specific to each class, or be tailored to a survey.  In the best case, the human integration is minimally invasive, and operates on features or predictions of existing networks.

**Comment by mdhamer** [comment]
Here are some of the types of cues (with examples) that we use while flying observational surveys in the Salish Sea during the winter. These aren't necessarily rules, but are hints that help the observer to quickly narrow down to potential species/taxa. Some of these are repeats from Ben's post but may help in thinking about additional processes that could be added to future models or post-model processing.

1. Species Co-Occurrence:
    -    Wigeon often sit with scaup to pick at vegetation the scaup bring to the surface  



2.  Species Flocking Patterns:
    -    Marbled Murrelets are often in pairs vs. Ancient Murrelets are often found in groups of 5-7+
    -    Common Murres often raft in long strings
    -    Pacific and Red-throated Loons are generally more gregarious than Common Loons and are typically found in larger flocks



3.  Species Habitat Associations:
    -    Common Mergansers are more common on freshwater than saltwater; reversed for Red-Breasted Merganser, but plenty of overlap
    -    Harlequin Ducks are typically found along mixed-sediment to rocky shorelines with shallow slopes and/or submerged rocky reefs
    -    Barrow's Goldeneye are more common in certain Salish Sea basins



4.  Male/Female Associations:
    -    Female counts of sexually dimorphic species are often grouped with the species of the closest male; males are generally easier to ID (Surf vs. White-winged Scoters; Common vs. Barrow's Goldeneye)



5.  Plumage Transitions:
    -    At the beginning of our winter survey period Pigeon Guillemot are in grey winter plumage, by the end of the survey many have transitioned to black and white breeding plumage



6.  Behavior:
    -    Western grebes won't likely be seen flying.... likely a loon species instead
